pkg/sink/sink.go

```diff
@@ -164,7 +160,7 @@ func (r Sink) processTrigger(t *triggersv1.EventListenerTrigger, request *http.R
 		log.Error(err)
 		return err
 	}
-	log.Info("params: %+v", params)
+	log.Info("params: %+v", params) // Is this useful without the context of the event??
```

Comment:
Hm, yeah, it probably needs some context...

Looks like we log some info earlier: https://github.com/tektoncd/triggers/blob/master/pkg/sink/sink.go#L89
```go
eventLog.Debugf("EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
		r.EventListenerName, r.EventListenerNamespace, eventID, string(event), request.Header)
```

Maybe we should change this `log.Info` into a `log.Debug`? Or turn that `log.Debugf` into a `log.Infof`.

Also, looks like this should be a `log.Infof` instead of `log.Info`

Reply:
Made this a log.Infof. Looks like we already decorate the logger with the Trigger name and the eventID which I think adds enough context (the log contains a triggers-id and event-id field):

```
{"level":"info","logger":"eventlistener","caller":"sink/sink.go:167","msg":"params: %+v[{gitrevision {string 1a1736ec3d7b03349b31218a2f2c572c7c7206d6 []}} {gitrepositoryurl {string https://gitlab.com/dibyom/triggers.git []}}]","knative.dev/controller":"eventlistener","/triggers-eventid":"ntd6w","/trigger":"gitlab-push-events-trigger"}
```